bundle-size/test/api.test.js

```diff
@@ -427,36 +427,42 @@ describe('bundle-size api', () => {
         baseBundleSizeFixture.content = Buffer.from(
           '{"dist/v0.js":12.34}'
         ).toString('base64');
-        const nocks = nock('https://api.github.com')
-          .get(
-            '/repos/ampproject/amphtml-build-artifacts/contents/' +
-              'bundle-size/5f27002526a808c5c1ad5d0f1ab1cec471af0a33.json'
-          )
-          .times(2)
-          .reply(404)
-          .get(
-            '/repos/ampproject/amphtml-build-artifacts/contents/' +
-              'bundle-size/5f27002526a808c5c1ad5d0f1ab1cec471af0a33.json'
-          )
-          .reply(200, baseBundleSizeFixture)
-          .patch('/repos/ampproject/amphtml/check-runs/555555', body => {
-            expect(body).toMatchObject({
-              conclusion: 'success',
-              output: {
-                title: 'Δ +0.00KB | no approval necessary',
-              },
+        const lastNetworkRequest = new Promise(resolve => {
+          const nocks = nock('https://api.github.com')
+            .get(
+              '/repos/ampproject/amphtml-build-artifacts/contents/' +
+                'bundle-size/5f27002526a808c5c1ad5d0f1ab1cec471af0a33.json'
+            )
+            .times(2)
+            .reply(404)
+            .get(
+              '/repos/ampproject/amphtml-build-artifacts/contents/' +
+                'bundle-size/5f27002526a808c5c1ad5d0f1ab1cec471af0a33.json'
+            )
+            .reply(200, baseBundleSizeFixture)
+            .patch('/repos/ampproject/amphtml/check-runs/555555', body => {
+              expect(body).toMatchObject({
+                conclusion: 'success',
+                output: {
+                  title: 'Δ +0.00KB | no approval necessary',
+                },
+              });
+              return true;
+            })
+            .reply(200, () => {
+              setTimeout(() => {
+                resolve(nocks);
+              }, 0);
             });
-            return true;
-          })
-          .reply(200);
+        }).then(nocks => nocks.done());
```

Comment:
Wouldn't this be clearer with `nocks.done()` as the last statement in the test?

Reply:
after `await lastNetworkRequest;`? Yeah, I can do that

Reply:
Oh wait, since nocks is returned with the promise resolve, I have to do it *with* that await

Reply:
Done, PTAL

I'll fix the broken test tomorrow